system-test/error-reporting.js

```diff
@@ -522,7 +522,8 @@ describe('error-reporting', function() {
       },
       extraConfig || {}
     );
-    errors = require('../src/index.js')(config);
+    const {Errors} = require('../src/index.js');
```

Comment:
Not sure if you're transpiling this js file, but I think this syntax doesn't work in Node 4

Reply:
Yes, good catch.  I'll update it.

Reply:
Fixed